hamlet.py

```diff
@@ -2,6 +2,11 @@
 # -*- coding: utf-8 -*-
 """This module does some pretty crazy math."""
 
+import task_02
```

Comment:
I think your circular import reference here may have been partly problematic. You shouldn't have needed to modify `hamlet.py` at all.